src/views/partials/floatingBanner.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AppConstants from '../../appConstants.js'
+import { getMessage } from '../../utils/fluent.js'
+
+/**
+ * @param {{ pathname: string; }} options
+ * @returns { string } banner
+ */
+export const floatingBanner = ({ pathname }) => {
+  const pageHasFloatingBanner = AppConstants.FLOATING_BANNER_PAGES?.split(',').includes(pathname)
+  console.log(pageHasFloatingBanner)
+  if (!pageHasFloatingBanner) {
+    return ''
+  }
+
+  return `
+    <link rel='stylesheet' href='/css/partials/floatingBanner.css' type='text/css'>
+    <script src='/js/partials/floatingBanner.js' type='module'></script>
+    <div class='floating-banner' data-type='${AppConstants.FLOATING_BANNER_TYPE ?? ''}' data-delay='${AppConstants.FLOATING_BANNER_DELAY ?? ''}' hidden>
```

Comment:
I was curious, but if I'm reading this correctly, the fallback delay of `''` won't mess up anything w/  `parseInt()`, `setInterval()`, or `setTimeout()` since it seems the `floatingBanner#init()` method will abort early if `data-type` or `data-delay` are falsey. 👍 

And can't see any obvious issues w/ a potential value like `data-delay="0"` which is falsey, but only as a number, not a string; I think... Although I can't see if it'd break if we put something non-numeric and unparseable by `parseInt(n, 10)` and it gets passes into `setTimeout()`... But now we're talking edges-on-edges.

https://github.com/mozilla/blurts-server/blob/66925414402b07e8a3bc9fb4b75396f1d6a78b6e/src/client/js/partials/floatingBanner.js#L13-L17